From achiang@hp.com  Tue Oct 27 11:25:08 2009
From: Alex Chiang <achiang@hp.com>
Date: Wed, 21 Oct 2009 21:45:20 -0600
Subject: Documentation: ABI: document /sys/devices/system/cpu/
To: randy.dunlap@oracle.com, greg@kroah.com
Cc: akpm@linux-foundation.org, linux-kernel@vger.kernel.org, rientjes@google.com
Message-ID: <20091022034520.13399.23985.stgit@bob.kio>


This interface has been around for a long time, but hasn't been
officially documented.

Document the top level sysfs directory for CPU attributes.

Signed-off-by: Alex Chiang <achiang@hp.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/ABI/testing/sysfs-devices-system-cpu |   12 ++++++++++++
 1 file changed, 12 insertions(+)

--- a/Documentation/ABI/testing/sysfs-devices-system-cpu
+++ b/Documentation/ABI/testing/sysfs-devices-system-cpu
@@ -1,3 +1,15 @@
+What:		/sys/devices/system/cpu/
+Date:		pre-git history
+Contact:	Linux kernel mailing list <linux-kernel@vger.kernel.org>
+Description:
+		A collection of both global and individual CPU attributes
+
+		Individual CPU attributes are contained in subdirectories
+		named by the kernel's logical CPU number, e.g.:
+
+		/sys/devices/system/cpu/cpu#/
+
+
 What:      /sys/devices/system/cpu/cpu*/cache/index*/cache_disable_X
 Date:      August 2008
 KernelVersion:	2.6.27
